src/test/java/uk/gov/hmcts/reform/userprofileapi/service/impl/UserProfileRetrieverTest.java

```diff
@@ -167,6 +184,38 @@ public void should_retrieve_Multiple_Profiles() {
         verify(auditRepository, times(2)).save(any());
     }
 
+    @Test
+    public void should_retrieve_Multiple_Profiles_RolesRequiredFalse() {
+        idamRolesInfo = new IdamRolesInfo(entity, HttpStatus.CREATED);
+
+        List<UserProfile> userProfiles = new ArrayList<>();
+
+        UserProfile up1 = UserProfileTestDataBuilder.buildUserProfile();
+        up1.setStatus(IdamStatus.ACTIVE);
+        userProfiles.add(up1);
+
+        UserProfile up2 = UserProfileTestDataBuilder.buildUserProfile();
+        up2.setStatus(IdamStatus.ACTIVE);
+        userProfiles.add(up2);
+
+        UserProfileIdentifier identifier = new UserProfileIdentifier(IdentifierName.UUID_LIST, Arrays.asList(UUID.randomUUID().toString(), UUID.randomUUID().toString()));
+
+        when(querySupplier.getProfilesByIds(identifier, true)).thenReturn(Optional.of(userProfiles));
+
+        List<UserProfile> userProfilesWithRoles = userProfileRetriever.retrieveMultipleProfiles(identifier, true, false);
+        assertThat(userProfilesWithRoles.size()).isEqualTo(2);
+
+        UserProfile getUserProfile1 = userProfilesWithRoles.get(0);
+
+        assertThat(getUserProfile1.getEmail()).isEqualTo(up1.getEmail());
+        assertThat(getUserProfile1.getFirstName()).isEqualTo(up1.getFirstName());
+        assertThat(getUserProfile1.getLastName()).isEqualTo(up1.getLastName());
+        assertThat(getUserProfile1.getRoles()).isEqualTo(up1.getRoles());
+        assertThat(getUserProfile1.getErrorMessage()).isEqualTo(up1.getErrorMessage());
+        assertThat(getUserProfile1.getErrorStatusCode()).isEqualTo(up1.getErrorStatusCode());
+        verify(querySupplier, times(1)).getProfilesByIds(identifier, true);
+    }
+
     @Test
     public void should_throw_404_when_no_profiles_found_in_db() {
         UserProfileIdentifier identifier = mock(UserProfileIdentifier.class);
```

Comment:
all tests in the class do not have test as prefix.. if this is a standard practice followed then it should be updated for all unit tests

Reply:
this has been addressed in the following commit: https://github.com/hmcts/rd-user-profile-api/pull/385/commits/8d2cf5fd8500a9e186c888746072467e7fcf8a3e